### PR TITLE
Add typescript definition

### LINKED ISSUE #293: Typescript support?
I can find the typescript support for 0.5.1 (@types/react-images) but not the latest 1.x version.

Any plan to support this?

### PR DESCRIPTION
**Pull request by raymondsze**
Fix #293

### CODE REVIEW

**Comment by lmerotta** [comment]
When will this be merged ?

**Comment by skurger** [comment]
Why has this not been merged?  Any reasons specifically?

**Comment by davwheat** [comment]
Sorry, I hadn't gotten around to this issue yet.

**Comment by alex-chuev** [comment]
@davwheat Please update npm package

**Comment by davwheat** [comment]
@alex-chuev Pushed beta update: 1.1.0-beta.2

**Comment by davedc** [comment]
@davwheat I think you have missed adding the declaration file on build.

**Comment by alex-chuev** [comment]
@davwheat Please add the declaration file in the build by updating the list of `files` in the `package.json` file

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "main": "lib/index.js",
   "jsnext:main": "dist/react-images.es.js",
   "module": "dist/react-images.es.js",
+  "types": "./index.d.ts",
```
Comment (doublejosh) [inline package.json]: Here's the bug.

**Pull request merged**